modules/ipfs-cpinner-client/src/auth-manager/types.ts

```diff
@@ -0,0 +1,16 @@
+export type LoginResponse = { accessToken: string, refreshToken: string }
+
+export type PersonalSign = (data: string) => Promise<string>
+
+export interface ClientKeyValueStorage {
+  get (key: string): Promise<string>
+  set (key: string, value: string): Promise<void>
+}
+
+export type DIDAuthConfig = {
+  did?: string
+  serviceUrl: string
+  serviceDid?: string // TODO: unused, if we verify challenge request we should use it to assert signer, otherwise remove it
```

Comment:
You're right, it was used before we changed the way we sign in the login process. Anyway, I think we should verify that the signer is the one we expect to be because is supposed that the DV client must trust in the DV service.

Reply:
Do we open an issue for this?

Reply:
Already opened, let's remove it for now #56